lib/api/api_test.go

```diff
@@ -36,10 +36,15 @@ import (
 )
 
 var (
-	confDir = filepath.Join("testdata", "config")
-	token   = filepath.Join(confDir, "csrftokens.txt")
+	confDir      = filepath.Join("testdata", "config")
+	token        = filepath.Join(confDir, "csrftokens.txt")
+	testEvLogger = events.NewLogger()
 )
 
+func init() {
+	go testEvLogger.Serve()
```

Comment:
An opportunity for events.NoopLogger that doesn't need a serve? Or do these tests rely on that working?

Reply:
As far as I see the tests don't rely on them working, but they start an api service which interacts with it. The types in events are concrete, not interfaces, so this would require new interfaces and two noop-types - I don't think it's worth the effort.